data/mods/Xedra_Evolved/recipes/inventor/armor.json

```diff
@@ -0,0 +1,160 @@
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 10 ] ],
+    "qualities": [ { "id": "SAW_M", "level": 2 }, { "id": "SAW_M_FINE", "level": 1 }, { "id": "GRIND", "level": 1 } ],
+    "proficiencies": [
+      { "proficiency": "prof_elec_soldering", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_metalworking", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_welding_basic", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_elec_circuits", "time_multiplier": 2, "skill_penalty": -100 }
+    ],
+    "components": [
+      [
+        [ "headgear", 1 ],
+        [ "helmet_army", 1 ],
+        [ "tac_fullhelmet", 1 ],
+        [ "tac_helmet", 1 ],
+        [ "firehelmet", 1 ],
+        [ "helmet_ball", 1 ],
+        [ "helmet_football", 1 ]
+      ],
+      [ [ "magnetron", 1 ] ],
+      [ [ "power_supply", 1 ] ]
+    ],
+    "time": "18 h",
+    "skill_used": "deduction",
+    "difficulty": 4,
+    "skills_required": [ "electronics", 4 ],
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_ARMOR"
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "inventor_backpack",
+    "using": [ [ "welding_standard", 20 ] ],
+    "proficiencies": [
+      { "proficiency": "prof_elec_soldering", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_metalworking", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_welding_basic", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_welding", "time_multiplier": 2, "skill_penalty": -100 }
+    ],
+    "qualities": [ { "id": "SAW_M", "level": 2 }, { "id": "SAW_M_FINE", "level": 1 }, { "id": "GRIND", "level": 1 } ],
+    "tools": [ [ [ "vac_pump", 300 ] ] ],
+    "components": [ [ [ "molle_medium_rucksack", 1 ], [ "molle_large_rucksack", 1 ] ], [ [ "scrap_aluminum", 30 ] ] ],
+    "time": "18 h",
+    "skill_used": "deduction",
+    "difficulty": 7,
+    "skills_required": [ "fabrication", 5 ],
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_ARMOR"
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "inventor_leg_weight",
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 10 ] ],
+    "qualities": [ { "id": "SAW_M", "level": 2 }, { "id": "SAW_M_FINE", "level": 1 }, { "id": "GRIND", "level": 1 } ],
+    "proficiencies": [
+      { "proficiency": "prof_elec_soldering", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_metalworking", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_welding_basic", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_welding", "time_multiplier": 2, "skill_penalty": -100 }
+    ],
+    "components": [
+      [ [ "tool_belt", 1 ], [ "police_belt", 1 ], [ "leather_belt", 1 ], [ "webbing_belt", 1 ] ],
```

Comment:
Add 12 pipes, 5 point harness item once that is added. Pipe fittings and a pressure gauge and combat/hiking boots.